{"uuid":"6566ca8e-0a69-45c3-9c8b-e8af324ddc91","historyId":"db85d92f26f592a1430427d790cb607d","fullName":"com.thoaikx.android.LoginTest.placeOrder","labels":[{"name":"package","value":"com.thoaikx.android.LoginTest"},{"name":"testClass","value":"com.thoaikx.android.LoginTest"},{"name":"testMethod","value":"placeOrder"},{"name":"parentSuite","value":"AndroidDevices"},{"name":"suite","value":"Device 2"},{"name":"subSuite","value":"com.thoaikx.android.LoginTest"},{"name":"host","value":"192.168.1.248.non-exists.ptr.local"},{"name":"thread","value":"34873@192.168.1.248.non-exists.ptr.local.TestNG-tests-2(15)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"placeOrder","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Appium HTTP server is not listening at http://0.0.0.0:4724/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.\nNode.js executable path: /usr/local/bin/node\nArguments: [/usr/local/lib/node_modules/appium/build/lib/main.js, --port, 4724, --address, 0.0.0.0, --log, /Users/mac/Documents/Appium-Android-IOS-Automation/logs/appium-android-4724.log, --relaxed-security]\nServer log: \u001B[38;5;112m[Appium]\u001B[0m Welcome to Appium v2.17.1\n\u001B[38;5;112m[Appium]\u001B[0m Non-default server args:\n\u001B[38;5;112m[Appium]\u001B[0m { port: \u001B[33m4724\u001B[39m, relaxedSecurityEnabled: \u001B[33mtrue\u001B[39m }\n\u001B[38;5;112m[Appium]\u001B[0m The autodetected Appium home path: /Users/mac/.appium\n\u001B[38;5;112m[Appium]\u001B[0m Attempting to load driver xcuitest...\n\u001B[38;5;112m[Appium]\u001B[0m Attempting to load driver uiautomator2...\n\u001B[38;5;112m[Appium]\u001B[0m Requiring driver at /Users/mac/.appium/node_modules/appium-uiautomator2-driver/build/index.js\n\u001B[38;5;112m[Appium]\u001B[0m Requiring driver at /Users/mac/.appium/node_modules/appium-xcuitest-driver/build/index.js\n","trace":"io.appium.java_client.service.local.AppiumServerHasNotBeenStartedLocallyException: Appium HTTP server is not listening at http://0.0.0.0:4724/ after 20000 ms timeout. Consider increasing the server startup timeout value and check the server log for possible error messages occurrences.\nNode.js executable path: /usr/local/bin/node\nArguments: [/usr/local/lib/node_modules/appium/build/lib/main.js, --port, 4724, --address, 0.0.0.0, --log, /Users/mac/Documents/Appium-Android-IOS-Automation/logs/appium-android-4724.log, --relaxed-security]\nServer log: \u001B[38;5;112m[Appium]\u001B[0m Welcome to Appium v2.17.1\n\u001B[38;5;112m[Appium]\u001B[0m Non-default server args:\n\u001B[38;5;112m[Appium]\u001B[0m { port: \u001B[33m4724\u001B[39m, relaxedSecurityEnabled: \u001B[33mtrue\u001B[39m }\n\u001B[38;5;112m[Appium]\u001B[0m The autodetected Appium home path: /Users/mac/.appium\n\u001B[38;5;112m[Appium]\u001B[0m Attempting to load driver xcuitest...\n\u001B[38;5;112m[Appium]\u001B[0m Attempting to load driver uiautomator2...\n\u001B[38;5;112m[Appium]\u001B[0m Requiring driver at /Users/mac/.appium/node_modules/appium-uiautomator2-driver/build/index.js\n\u001B[38;5;112m[Appium]\u001B[0m Requiring driver at /Users/mac/.appium/node_modules/appium-xcuitest-driver/build/index.js\n\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:192)\n\tat com.thoaikx.base.BaseMobileAndroid.setUpAndroid(BaseMobileAndroid.java:58)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)\n\tat org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:810)\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:577)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:848)\n\tat org.testng.TestRunner.run(TestRunner.java:621)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:443)\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:481)\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\n\tat java.base/java.lang.Thread.run(Thread.java:840)\nCaused by: io.appium.java_client.service.local.AppiumServerAvailabilityChecker$ConnectionTimeout: ConnectionTimeout\n\tat io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:72)\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.ping(AppiumDriverLocalService.java:144)\n\tat io.appium.java_client.service.local.AppiumDriverLocalService.start(AppiumDriverLocalService.java:185)\n\t... 27 more\nCaused by: java.net.ConnectException: Connection refused\n\tat java.base/sun.nio.ch.Net.pollConnect(Native Method)\n\tat java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)\n\tat java.base/sun.nio.ch.NioSocketImpl.timedFinishConnect(NioSocketImpl.java:547)\n\tat java.base/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:602)\n\tat java.base/java.net.Socket.connect(Socket.java:633)\n\tat java.base/sun.net.NetworkClient.doConnect(NetworkClient.java:178)\n\tat java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:533)\n\tat java.base/sun.net.www.http.HttpClient.openServer(HttpClient.java:638)\n\tat java.base/sun.net.www.http.HttpClient.<init>(HttpClient.java:281)\n\tat java.base/sun.net.www.http.HttpClient.New(HttpClient.java:386)\n\tat java.base/sun.net.www.http.HttpClient.New(HttpClient.java:408)\n\tat java.base/sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:1323)\n\tat java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1256)\n\tat java.base/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1142)\n\tat java.base/sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:1071)\n\tat io.appium.java_client.service.local.AppiumServerAvailabilityChecker.connectToUrl(AppiumServerAvailabilityChecker.java:79)\n\tat io.appium.java_client.service.local.AppiumServerAvailabilityChecker.waitUntilAvailable(AppiumServerAvailabilityChecker.java:61)\n\t... 29 more\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"deviceIndex","value":"1"}],"start":1744447709337,"stop":1744447709338}